### PR TITLE
Updated to 1.20.1 and fixed formatting issue

### PR DESCRIPTION
**Pull request by Greymagic27**
Tested locally and plugin appeared green when I did /plugins. Seemed to operate correctly.

See internal channel for more info

### CODE REVIEW

File: advanced-achievements-plugin/src/main/java/com/hm/achievement/listener/statistics/AbstractRateLimitedListener.java [hunk 1]
```diff
@@ -119,7 +124,7 @@ private boolean isInCooldownPeriod(Player player, int slotNumber) {
 	 */
 	private void displayActionBarMessage(Player player, long timeToWait) {
 		String timeWithOneDecimal = String.format("%.1f", (double) timeToWait / 1000);
-		String message = "&o" + StringUtils.replaceOnce(langStatisticCooldown, "TIME", timeWithOneDecimal);
+		String message = "\u00A7o" + StringUtils.replaceOnce(langStatisticCooldown, "TIME", timeWithOneDecimal);
```
Comment (Greymagic27) [inline advanced-achievements-plugin/src/main/java/com/hm/achievement/listener/statistics/AbstractRateLimitedListener.java:127]: This is the formatting change.

Rest of changes in this file are just from when I saved it and auto formatting took over

File: pom.xml [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<java.version>1.8</java.version>
+		<java.version>17</java.version>
```
Comment (Greymagic27) [inline pom.xml:19]: Updated to java 17

File: pom.xml [hunk 3]
```diff
@@ -100,13 +100,12 @@
...
 				<artifactId>commons-text</artifactId>
-				<version>1.9</version>
+				<version>1.10.0</version>
 			</dependency>
 			<dependency>
 				<groupId>org.spigotmc</groupId>
 				<artifactId>spigot-api</artifactId>
-				<version>1.17-R0.1-SNAPSHOT</version>
-				<type>jar</type>
+				<version>1.20.1-R0.1-SNAPSHOT</version>
```
Comment (Greymagic27) [inline pom.xml:108]: Updated to 1.20.1 spigot

File: advanced-achievements-plugin/pom.xml [hunk 4]
```diff
@@ -89,56 +89,67 @@
...
 		</dependency>
 		<dependency>
 			<groupId>org.apache.commons</groupId>
 			<artifactId>commons-text</artifactId>
+			<version>1.10.0</version>
 		</dependency>
 		<dependency>
 			<groupId>org.spigotmc</groupId>
 			<artifactId>spigot-api</artifactId>
+			<version>1.20.1-R0.1-SNAPSHOT</version>
```
Comment (Greymagic27) [inline advanced-achievements-plugin/pom.xml:112]: Updated to 1.20.1 spigot

**Comment by mrfloris** [comment]
Thank you for the help, this gets us to keep moving forward with this originally-abandoned plugin.

**Pull request merged**